raiden/src/filter_expression/mod.rs

```diff
@@ -171,6 +172,21 @@ impl<T> FilterExpressionBuilder<T> for FilterExpressionFilledOrWaitOperator<T> {
                     attr_values,
                 )
             }
+            FilterExpressionTypes::In(attributes) => {
+                let placeholders = attributes
+                    .iter()
+                    .map(|(placeholder, _)| placeholder.clone())
+                    .collect::<Vec<_>>()
+                    .join(",");
```

Comment:
The placeholders are composed individually as `:value0,:value1,...` because dynamodb list data type (`L`) is not supported to work as a IN filter parameter.